app/backend/aquifers/models.py

```diff
@@ -0,0 +1,339 @@
+"""
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import uuid
+"""
+import datetime
+"""
+from django.utils import timezone
+from django.db import models
+from gwells.models import AuditModel
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+class AquiferMaterial(AuditModel):
+    """
+    Material choices for describing Aquifer Material
+
+    aquifer_materials
+    -------------------
+    Bedrock
+    Gravel
+    Sand
+    Sand and Gravel
+    """
+    code = models.CharField(primary_key=True, max_length=10, db_column='aquifer_material_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_material_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Material Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class AquiferSubtype(AuditModel):
+    """
+    Subtypes of  for describing Aquifer Material
+
+From Trello ticket
```

Comment:
Please insert tabs to align with other comments

Reply:
Done.. although this is a placeholder comment until I can create the *.json file to populate the code table.